<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_initsetting</class>
 <widget class="QDialog" name="Dialog_initsetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>140</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>20</y>
     <width>213</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_1">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>初始经度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>初始纬度:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>初始高度:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit_Lontitude">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Latitude">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_Altitude">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>航路点序列(经纬高)：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveTargeNode">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>370</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载目标序列</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>保存当前配置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_loadSettings">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>加载配置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>270</y>
     <width>189</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>串口：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_serialPortName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_hide1" native="true">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>160</y>
     <width>241</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="lineEdit_Gama">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>-114</x>
      <y>16</y>
      <width>145</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>初始航迹方位角:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>-114</x>
      <y>50</y>
      <width>145</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="text">
     <string>初始航迹俯仰角:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Chi">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>18</y>
      <width>103</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>265</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>初始速度:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_V"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Debug">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_targetNodes">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>341</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_targetIP">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_targetPort">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>接收机IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>90</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>接收机端口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_saveTargeNode</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_initsetting</receiver>
   <slot>On_pushButton_saveTargeNode_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>579</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>612</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_saveSettings</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_initsetting</receiver>
   <slot>On_pushButton_saveSettings_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>421</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_loadSettings</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_initsetting</receiver>
   <slot>On_pushButton_loadSettings_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Debug</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_initsetting</receiver>
   <slot>On_pushButton_Debug_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>947</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>877</x>
     <y>535</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_initsetting</receiver>
   <slot>On_ok_button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>On_ok_button_clicked()</slot>
  <slot>On_pushButton_saveTargeNode_clicked()</slot>
  <slot>On_pushButton_saveSettings_clicked()</slot>
  <slot>On_pushButton_loadSettings_clicked()</slot>
  <slot>On_pushButton_Debug_clicked()</slot>
 </slots>
</ui>
